templates/includes/footer.html

```diff
@@ -18,6 +18,12 @@ <h3 class="p-heading--four">MAAS</h3>
                 <li class="p-list__item">
                     <a href="/docs">Docs</a>
                 </li>
+		<li class="p-list__item">
```

Comment:
Why so un-indented? Why would you do such a thing?

Reply:
I used Vim lol